scripts/initialize/main.go

```diff
@@ -147,8 +151,10 @@ func main() {
 		log.Fatal("unable to parse role descriptions file", err)
 	}
 
-	dsn := "postgres://" + dbUser + ":" + dbPassword + "@" + dbAddr + "/" + dbName + "?sslmode=disable"
```

Comment:
We should still get the DSN from these variables in case we have not provided the DSN env value.